src/persist-client/src/read.rs

```diff
@@ -128,8 +128,6 @@ where
                     // might get a key reference that that blob isn't returning yet.
                     // Keep trying, it'll show up. The deadline will eventually bail
                     // us out of this loop if something has gone wrong internally.
-                    //
-                    // TODO: This should increment a counter.
```

Comment:
we're going to have to take a holistic look at this when we do metrics anyway, so not really useful in the interim